docs/articles/documentation/guides/basic-guides/select-page-elements.md

```diff
@@ -136,7 +136,7 @@ This selector does the following:
 </html>
 ```
 
-If a selector matches multiple elements, the subsequent methods return results for all the elements. For instance, the following selector returns the child nodes of all `<div>` tags on the page:
+If a selector matches multiple elements, the subsequent methods return results for all the elements. The following selector returns the child nodes of all `<div>` tags on the page, including their children and so on:
```

Comment:
```suggestion
If a selector matches multiple elements, the subsequent methods return results for all elements. The following selector returns the child nodes of all `<div>` tags on the page, including their children:
```

Reply:
Doesn't this imply that only parents and their immediate children are affected? Because this actually affects their children and their children and their children until we reach the root.

Reply:
I see now. How about "including their children and their decendants"?

Or maybe simply "including their decendants"?
Let me know what you think.

Reply:
It thinks this works well:

> If a selector matches multiple elements, the subsequent methods return results for all elements. The following selector returns the child nodes of all `<div>` tags on the page, including their children and their descendants: